guides/include-cleaner.md

```diff
@@ -25,13 +16,51 @@ This means:
 This is opinionated, and stricter than "will it compile". It implies adding
 `#includes` that the compiler does not strictly need. Some `#includes` that
 satisfy the compiler should be replaced with more specific ones.
+If your codebase does not broadly aim to follow this style, you should disable
+these features.
```

Comment:
Should we change the negative statement to the positive one? I.e., "If your codebase broadly _aims_ to follow this style, you should _enable_ these features".

Reply:
Well, that's a stronger claim! Sure, why not :-)